flyway-core/src/test/resources/migration/dbsupport/sqlserver/sql/assembly/V1__assembly.sql

```diff
@@ -0,0 +1,59 @@
+WITH PERMISSION_SET = SAFE /* SAFE means that the assembly will not be allowed to access external resources such as
```

Comment:
This huge hexadecimal value is the assembly DLL file content.

They are some obvious safety concerns when integrating binary content in a code repo. However the involved risk is here relatively low since it is only for test purpose, and the configured safety level should forbid to do anything nasty outside of the Database...